### ISSUE TITLE
Extract Watson Assistant training data, for use in bootstrapping a STT accuracy test

### ISSUE THREAD

**Issue by andrewrfreed** [IBM/watson-tts-python#9]
Some users have limited STT testing data.  A quick augmentation or bootstrapping of STT test data is to get recordings from Watson TTS.  (These recordings will NOT be representative of actual users - the voices are still "synthetic" - but it is better than not having any test data at all.)

Need two updates:
1) Ability to extract expected "inputs" from Watson Assistant (intent and entity examples)
2) Optionally, create a ground truth / reference file for [watson-stt-wer-python](https://github.com/IBM/watson-stt-wer-python) when doing synthesis.

Part 1 because we expect Watson Assistant is trained on a variety of representative utterances we want STT to hear.
Part 2 because we know the filename and the exact text spoken, we should record it!

**Comment by andrewrfreed** [comment]
merged
